monkey/monkey_island/cc/services/config_schema.py

```diff
@@ -13,84 +13,96 @@
                     "enum": [
                         "SmbExploiter"
                     ],
-                    "title": "SMB Exploiter"
+                    "title": "SMB Exploiter",
+                    "attack_techniques": ["T1110", "T1210", "T1075"]
```

Comment:
T1075 shouldn't be used here. The right thing is to add a config value which determines whether to use PTH and it should be linked to it.
In any exploit that we use pass the hash it should do so only if it's enabled by config

Reply:
This will be solved on another branch